download_and_create_reference_datasets/v03/write_combined_reference_ht.py

```diff
@@ -9,35 +11,36 @@
 from hail_scripts.reference_data.config import GCS_PREFIXES, AccessControl
 from hail_scripts.utils.hail_utils import write_ht
 
-COMBINED_REFERENCE_HT_PATH = (
-    'combined_reference/combined_reference.GRCh{genome_version}.ht'
-)
+COMBINED_REFERENCE_HT_PATH = 'reference_datasets/combined.ht'
 DATASETS = [
     'cadd',
     'clinvar',
     'dbnsfp',
     'eigen',
     'exac',
-    'geno2mp',
     'gnomad_genomes',
     'gnomad_exomes',
     'mpc',
     'primate_ai',
     'splice_ai',
     'topmed',
-    'gnomad_genome_coverage',
-    'gnomad_exome_coverage',
 ]
 
 
-def run(environment: str, genome_version: str, dataset: str):
+def run(environment: str, genome_version: str, dataset: str | None):
+    hl._set_flags(  # noqa: SLF001
+        no_whole_stage_codegen='1',
+    )  # hail 0.2.78 hits an error on the join, this flag gets around it
```

Comment:
It would probably be good to test if this is still needed at some point

Reply:
it definitely is, though I don't know why.  I was missing this line while testing at some point, and we failed at least a couple of times for strange reasons.  I added this line back in and we got way faster and did not fail.

Reply:
Well that is good to know!